src/subapps/projects/components/ActionButton.tsx

```diff
@@ -20,8 +20,8 @@ const ActionButton: React.FC<{
   }
 
   const onClickActionButton = (event: any) => {
```

Comment:
lets make sure to use the proper typing. You should be able to see the type by hovering over the onClick field of the component.